<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadView</class>
 <widget class="QWidget" name="LoadView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>484</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Recent Media</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="m_playFileButton">
       <property name="text">
        <string comment="@action:button">Play File…</string>
       </property>
       <property name="icon">
        <iconset theme="folder">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_playStreamButton">
       <property name="text">
        <string comment="@action:button">Play Stream…</string>
       </property>
       <property name="icon">
        <iconset theme="document-open-remote">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_playDiskButton">
       <property name="text">
        <string comment="@action:button">Play Disc</string>
       </property>
       <property name="icon">
        <iconset theme="media-optical">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="m_vThumb" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="m_clearRecent">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Clear Recent Media</string>
     </property>
     <property name="accessibleName">
      <string>Clear Recent Media</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="icon">
      <iconset theme="edit-clear-all">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="RecentlyPlayedList" name="m_recentlyPlayed">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="accessibleDescription">
      <string>Recent Media List</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RecentlyPlayedList</class>
   <extends>QListWidget</extends>
   <header>recentlyPlayedList.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
